examples/interactive/interactive_test.go

```diff
@@ -89,7 +89,7 @@ func createData() (perr error) {
 // TestReadOnlyThanosSetup runs read only Thanos setup that has data from `maxTimeFresh - 2w` to `maxTimeOld`, with extra monitoring and tracing for full playground experience.
 // Run with test args `-test.timeout 9999m`.
 func TestReadOnlyThanosSetup(t *testing.T) {
-	t.Skip("This is interactive test - it will until you will kill it or curl 'finish' endpoint. Uncomment and run as normal test to use it!")
+	t.Skip("This is interactive test - it will until you will kill it or curl 'finish' endpoint. Comment and run as normal test to use it!")
```

Comment:
> t.Skip("This is interactive test - it will until you will kill it or curl 'finish' endpoint. Comment and run as normal test to use it!")

... it will run until...

Reply:
@bwplotka sorry for the delay, I shall open a PR fixing this soon.